examples/scripts/DFN.py

```diff
@@ -31,7 +31,7 @@
 # solve model
 t_eval = np.linspace(0, 3600, 100)
 solver = pybamm.CasadiSolver(atol=1e-6, rtol=1e-3)
-solution = solver.solve(model, t_eval)
+solution = solver.solve(model, t_eval, inputs={"eps_n": 0.3})
```

Comment:
I think it would be better to keep the DFN example without inputs, we can have a separate example for this.

Reply:
yes, forgot to remove this